### PR TITLE
Implemented interface and tests for Studentized Range distribution

### PR DESCRIPTION
**Pull request by BioTurboNick**
Relies on this PR for StatsFuns.jl: https://github.com/JuliaStats/StatsFuns.jl/pull/66

Uses R's functions for cdf and quantile, no pdf implementation. Tests implemented with relaxed matching tolerance (R implementation reports accuracy to only 4th decimal place).

### CODE REVIEW

**Comment by BioTurboNick** [comment]
StatsFuns.jl PR was approved and tagged, tests should pass now.

**Comment by simonbyrne** [comment]
The required StatsFuns version should be bumped to 0.8.0

File: REQUIRE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 julia 0.7
 PDMats 0.9.0
-StatsFuns 0.7.1
+StatsFuns 0.8.0
```
Comment (matbesancon) [inline REQUIRE]: we shouldn't use patch numbers there, `0.8` instead of `0.8.1`

**Comment by matbesancon** [comment]
one minor change and it's good to merge for me

**Comment by matbesancon** [comment]
thanks @BioTurboNick

**Pull request merged**